openc_bot.gemspec

```diff
@@ -49,4 +49,5 @@ Gem::Specification.new do |gem|
   # gem.add_development_dependency "perftools.rb"
   gem.add_development_dependency "byebug", "~> 10.0"
   gem.add_development_dependency "rspec", "~> 3.8"
+  gem.add_development_dependency "rubocop-rspec", "~> 1.30"
```

Comment:
I think if somebody removes this gem and finds that the CI breaks, the error messages will make it pretty clear what needs to happen. Adding `rubocop` explicitly means the dependencies of both gems need to be investigated before they are removed.

However, since you think it's good practice I have added it.

Reply:
I think the main issue is that we have a _hard_ dependency on both `rubocop` and `rubocop-rspec`, ie we are writing code (or config in this case, I suppose) aimed at a specific version. So we will want control over the version of `rubocop` we use, rather than leaving it completely up to `rubocop-rspec`, otherwise `rubocop-rspec` can upgrade `rubocop` to something we don't want.